lib/main_common.pm

```diff
@@ -568,15 +568,18 @@ sub load_jeos_tests {
     load_boot_tests();
     loadtest "jeos/firstrun";
     loadtest "jeos/record_machine_id";
+    loadtest "console/system_prepare" if is_sle;
     loadtest "console/force_scheduled_tasks";
     unless (get_var('INSTALL_LTP')) {
-        loadtest "jeos/grub2_gfxmode";
+        loadtest "jeos/grub2_gfxmode" if get_var('CONTAINER_RUNTIME');
         loadtest "jeos/diskusage";
         loadtest "jeos/build_key";
-        loadtest "console/prjconf_excluded_rpms";
+        loadtest "console/prjconf_excluded_rpms" if get_var('CONTAINER_RUNTIME');
```

Comment:
same question

Reply:
same reason :)

Reply:
can you also keep this line as it was? Same reason as for `loadtest "jeos/grub2_gfxmode" if get_var('CONTAINER_RUNTIME');`

Reply:
*this change is ready for review*